extensions/countdown-timer-in-c#.md

```diff
@@ -0,0 +1,29 @@
+# Streamer.bot Countdown Timer Import Code
```

Comment:
Need to add this to the top of the page 
---
title: Countdown Timer in C# (by rondhi)
description: Create a countdown timer with Streamer.bot.
published: true
date: 2022-05-18T02:56:07.033Z
tags: 
editor: markdown
dateCreated: 2022-05-17T00:31:26.288Z
---